doc/content/getting-started/migrating-from-networks/import-devices.md

```diff
@@ -3,20 +3,52 @@ title: Import End Devices in The Things Stack
 weight: 20
 ---
 
-{{< cli-only >}}
+To import your devices, you need an application in {{% tts %}}. This can be done by following instructions for [Creating an Application]({{< ref "integrations/adding-applications" >}}).
+
+To import devices, use the `devices.json` file you created in the previous step.
+
+{{< tabs/container "Console" "CLI" >}}
+
+{{< tabs/tab "Console" >}}
+
+Open the application you created and click the button **Import end devices**
+
+{{< figure src="../import-end-devices.png" alt="import devices" >}}
+
+Select `The Things Stack JSON` as the **Format** and upload the `devices.json` file.
+
+{{< figure src="../upload-file.png" alt="upload devices.json file" >}}
+
+Wait for the devices to be successfully imported. In case any device fails, you see a relevant error message in the console.
```

Comment:
Nit; we refer to them as "end devices" in the rest of the documentation, so better do this here, and in other places also.